cipher_suite.go

```diff
@@ -28,6 +28,10 @@ const (
 	TLS_PSK_WITH_AES_128_GCM_SHA256 CipherSuiteID = 0x00a8
 )
 
+var (
+	_ = allCipherSuites() // Necessary until this function isn't only used by Go 1.14
```

Comment:
This makes me unhappy but otherwise CI currently breaks since it considers `allCipherSuites()` to be dead code